tests/integration/targets/hcloud_certificate_info/defaults/main/main.yml

```diff
@@ -1,4 +1,4 @@
 # Copyright: (c) 2019, Hetzner Cloud GmbH <info@hetzner-cloud.de>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 ---
-hcloud_certificate_name: "always-there-cert"
```

Comment:
Can we delete the certificate after merging or are there any other tests using it?

Reply:
The target is the only place where this cert is used, in addition it is created and destroyed on each run. This might have been a source of race between parallel tests, as they all used the same cert name.